src/js/controllers/walletHome.js

```diff
@@ -27,6 +27,8 @@ angular.module('copayApp.controllers').controller('walletHomeController', functi
   this.showScanner = false;
   this.addr = {};
   this.lockedCurrentFeePerKb = null;
+  self.paymentExpired = true;
```

Comment:
Please, use `self` only wherever necessary. Take a look at this example: https://discourse.codecombat.com/t/javascript-this-vs-self/3143